app/controllers/alchemy/pages_controller.rb

```diff
@@ -231,7 +241,8 @@ def page_etag
     def render_fresh_page?
       !cache_page? || stale?(etag: page_etag,
         last_modified: @page.published_at,
-        public: !@page.restricted)
+        public: !@page.restricted,
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.